src/Security/Items.cs

```diff
@@ -1733,4 +1733,65 @@ public SecurityException (SecStatusCode code) : base (ToMessage (code))
 		{
 		}
 	}
+
+	public partial class SecKeyParameters : DictionaryContainer {
+		// For caching, as we can't reverse it easily.
+		SecAccessControl _secAccessControl;
+
+		public SecAccessControl AccessControl {
```

Comment:
that's missing an availability attribute, it's iOS8 (double check)
so not available in our minimum baseline of iOS 6